imsv-docs-docusaurus/openapi/endpoints/kyc/get-kyc-profile.yaml

```diff
@@ -1,15 +1,21 @@
 tags:
   - kyc
 summary: Get a KYC profile
-description: Get a KYC profile by account id if it exists. 
+description: Get a KYC profile by account id if it exists.
 parameters:
   - name: accountId
     in: path
-    description: Id of the account.
+    description: Id of the cardholder or identity account.
     example: 225d85e65495722bf6517ea0ba0d6f56
     required: true
     schema:
       type: string
+  - name: withPersonalDetails
```

Comment:
```suggestion
  - name: includePersonalDetails
```

Reply:
so, do you want me to change the API contract in amethyst? the reason I opted for withPersonalDetails is because it sounds more eloquent, like **get profile** ... with personal details, but I am ok with either approach

Reply:
I also have mixed feelings about this new convention. `withSmth` pattern looks and feels great as a function name that can be chained. But here we're dealing with query parameters, not functions. A request to this API would look like `GET http://amethyst/get-kyc-profie?withPersonalDetails=true` (I'm messing up the base URL but you know what I mean). And when we have something that looks like a value assignment statement, `includePersonalDetails=true` reads better than `withPersonalDetails=true`.

So if I had to vote on this I'd say yes, let's change the contract in amethyst and update parameter name.

Reply:
https://github.com/immersve/amethyst/pull/2100/files